yal/askaquestion.py

```diff
@@ -502,16 +516,57 @@ async def state_handle_timeout_response(self):
         error, fields = await rapidpro.get_profile(whatsapp_id)
         if error:
             return await self.go_to_state("state_error")
-        data = {
-            "feedback_survey_sent": "",
-        }
+        self.save_metadata("feedback_timestamp", "")
+        data = {"feedback_survey_sent": "", "feedback_timestamp": ""}
         error = await rapidpro.update_profile(whatsapp_id, data)
         if error:
             return await self.go_to_state("state_error")
 
         timeout_type_sent = fields.get("feedback_type")
+        self.save_metadata("feedback_type", timeout_type_sent)
```

Comment:
Do we ever use `feedback_type` from the metadata?

Reply:
This is just specifically for the AAQ flow.

I didn't want to make too many changes to how the flows work, and this flow works slightly differently to others. Instead of doing the routing to the correct state in the main flow, AAQ routes all types to one state, and that state sends the user to the correct state depending on the feedback type.

I didn't want to change that much, but I did need to now be able to store that and use it later, so that the error state could also route to the correct state (or I guess I could've created 2 error states). So this is only used so that when you say "go back to survey" in the error state, it knows which state to send you to.

Reply:
ah ok! no worries then 👍